package.json

```diff
@@ -134,6 +134,11 @@
           "default": true,
           "description": "%ext.config.removeExtensions%"
         },
+        "sync.sortAlphabetically": {
```

Comment:
can you put it as global settings instead of env. settings.

Reply:
How can i do that ?

Reply:
What about this comment ?

Reply:
`sortAlphabetically` needs to be moved into 

https://github.com/shanalikhan/code-settings-sync#global-settings

In this file
https://github.com/shanalikhan/code-settings-sync/blob/master/src/models/customConfig.model.ts